### ISSUE TITLE
My new tweet shows up twice in Pino

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#39]
```
What steps will reproduce the problem?
-Sending a new tweet

What is the expected output? What do you see instead?
-My new tweet shows up twice in Pino, but not on Twitter.com.

What version of the product are you using? On what operating system?
-Pino 0.1.1-karmic~ppa1
-Ubuntu 9.10 (64-bit)

Please provide any additional information below.
-I think this is the first time it happend.
```

Original issue reported on code.google.com by `58desi...@gmail.com` on 12 Jan 2010 at 5:09

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 12 Jan 2010 at 11:03
- Changed state: **Accepted**
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Closed by cb9e2c3a53 revision. (I hope).
```

Original comment by `tro...@gmail.com` on 27 Jan 2010 at 4:24
- Changed state: **Fixed**
